lib/widgets/actions.dart

```diff
@@ -16,101 +16,111 @@ import '../model/narrow.dart';
 import 'dialog.dart';
 import 'store.dart';
 
-Future<void> markNarrowAsRead(
+Future<bool> markNarrowAsRead(
```

Comment:
This might worth some dart doc explaining what it does and the meaning of the return value.